ngraph/core/src/op/binary_convolution.cpp

```diff
@@ -73,8 +73,18 @@ void op::v1::BinaryConvolution::validate_and_infer_types()
     // TODO: Add NodeValidationCheck to filters et once u1 is supported in nGraph Python API
     // (#49517)
```

Comment:
This ticket is closed. Should we remove the comment ?

Reply:
The ticket was closed as the scope was extended to other types. I have updated the ticket in the comment